### ISSUE TITLE
Support Salesforce Apex

### ISSUE THREAD

**Issue by egraether** [CoatiSoftware/CoatiBugTracker#333]
https://developer.salesforce.com/page/Apex

**Comment by JodieM** [comment]
Yes please, this would be a good way to understand complex Salesforce Apex code.

**Comment by bakneo** [comment]
I agree!! Salesforce is getting more and more used especially it is a spreading CRM platform

**Comment by egraether** [comment]
Since we cannot keep up with language requests, we created the [SourcetrailDB](https://github.com/CoatiSoftware/SourcetrailDB) library to allow anyone to write custom indexers and extend language support for Sourcetrail.

If you are interested in writing an indexer for Apex, please let us know!
